VK/VK/FriendPhotoScreen/Cell/PhotoCollectionViewCell.swift

```diff
@@ -16,23 +23,39 @@ final class PhotoCollectionViewCell: UICollectionViewCell {
 
     private var tapHandler: TapHandler?
     private var selectedFriend: NetworkUnit?
```

Comment:
```suggestion
    private var selectedFriend: NetworkUnit?
```
название у типа и объекта должны совпадать